src/nvim/options.lua

```diff
@@ -91,11 +91,11 @@ return {
     {
       abbreviation = 'al',
       defaults = { if_true = 224 },
-      enable_if = false,
```

Comment:
Should `false` be removed from the type annotation of `enable_if` now?

Reply:
You still haven't made this change.

Reply:
@famiu Did you seem this comment?

Reply:
> @famiu Did you seem this comment?

Oh sorry I missed it